### ISSUE TITLE
Website Design

### ISSUE THREAD

**Issue by blakeembrey** [typings/discussions#5]
It'd be nice to have a typings website that can point people to getting started and searching and reading about how to write typings and use TypeScript.

Some resources:
- https://github.com/typings/api  - The API for searching for typings and rendering results
- https://sebastian-lenz.github.io/ - A design for DefinitelyTyped, looks great
- http://definitelytyped.org/ - The current DefinitelyTyped website
- http://npmjs.org/ - The node package manager website (some ideas like most installed and recently updated can eventually be incorporated)
- http://bower.io/search/ - The Bower package search page (very simple, might be an easy starting point)

**Comment by senhongo** [comment]
@blakeembrey 
I'm probably biting off more than I can chew but I'm interested. How can I help?

**Comment by blakeembrey** [comment]
@SenHeng That would be amazing! I haven't had time to start anything yet, but if you wanted I can create a repo or a skeleton to work against? What's the easiest way for you?

**Comment by senhongo** [comment]
@blakeembrey 
We could start with an empty repo, then start figuring out what needs to be in the site?

From the resources you've linked in the first post, I guess you'd like for there to be some kind of discovery feature?

**Comment by blakeembrey** [comment]
Yes :smile: There the search API for this: https://api.typings.org/search. I'll create the repo and add you to it.

**Comment by senhongo** [comment]
@blakeembrey 
Thanks for adding me to the group.
I have a few ideas and I'll try putting some up next week. Is there any way I could circulate ideas with you or the other members? Some kind of a group chat/channel thing?
Or do I just put them up as issues in the new repo?

**Comment by blakeembrey** [comment]
Of course, there's https://gitter.im/typings where we can chat. You're also welcome to create issues or comments too :smile:

**Comment by blakeembrey** [comment]
Oh, I did make an issue asking Sebastian about his theme: https://github.com/sebastian-lenz/sebastian-lenz.github.io/issues/1

**Comment by hiroppy** [comment]
@blakeembrey  What are the key color and the sub color of `typings`?

**Comment by blakeembrey** [comment]
I can't say I've done any design work for @typings, so you're welcome to reuse the images I currently use (basically from https://unsplash.com/search?utf8=%E2%9C%93&keyword=type+writer&button=) and come up with colors too. I do like the color and design of Sebastian's work.

**Comment by hiroppy** [comment]
Thank you for the information :smile: 
I think key color is near `#111`.

**Comment by hiroppy** [comment]
@blakeembrey I made a draft(only root page). How do you think?
http://abouthiroppy.github.io/typings-website-prototype/

**Comment by blakeembrey** [comment]
Nice, we have some work to go on styling it still :+1: Have you considered grabbing a template and going from there? I particularly like (as noted) Sebastian's template. Since it is MIT, we could re-use that with proper attribution. I'd be happy to port it across and we can collaborate on tweaking it.

**Comment by abouthiroppy** [comment]
Oh, l'm sorry😩 I forgot the template. 
I agree to your opinion👍

**Comment by warren-nixon** [comment]
For those interested, you can find the original discussion about @sebastian-lenz's website and its design at DefinitelyTyped/definitelytyped.github.io/issues/14.

**Comment by ghost** [comment]
I would like to volunteer, too, since I'm already prototyping a small website that has some specific functionality for typings. What is the status-quo on this?

**Comment by blakeembrey** [comment]
I don't think any work has been done on it.

**Comment by ScorpioCoding** [comment]
And is the site completed?

**Comment by demurgos** [comment]
Hi,
The website never got completed. Now that `typings` is only in maintenance (since the release of Typescript 2), I don't think we need a website anymore.
